data/lib/compat/compat.lua

```diff
@@ -1005,3 +1005,33 @@ end
 function Guild.removeMember(self, player)
 	return player:getGuild() == self and player:setGuild(nil)
 end
+
+function doSetMonsterOutfit(cid, name, time)
+	local creature = Creature(cid)
+	if not creature then
+		return false
+	end
+	return creature:setMonsterOutfit(name, time)
+end
+
+function doSetItemOutfit(cid, item, time)
+	local creature = Creature(cid)
+	if not creature then
+		return false
+	end
+	return creature:setItemOutfit(item, time)
+end
```

Comment:
Most short functions are written like this in compat:
```lua
function doSetItemOutfit(cid, item, time) local c = Creature(cid) return c ~= nil and creature:setItemOutfit(item, time) end
function doSetMonsterOutfit(cid) local c = Creature(cid) return c ~= nil and creature:setMonsterOutfit(name, time) end
```
And you could do the same to save a few extra lines for the two above.